syntax/embed.php

```diff
@@ -48,6 +49,9 @@ public function handle($match, $state, $pos, Doku_Handler $handler)
         if (preg_match('/\b(left|right|center)\b/', $params, $matches)) {
             $data['align'] = $matches[1];
         }
+        if (preg_match('/\|(.*)/', $params, $matches)) {
+            $data['title'] = $matches[1];
+        }
```

Comment:
I think the title needs to be separated from the rest of the options first. Otherwise the following would be left aligned:

`<diagram |I left my keys at home>`

Reply:
Oh, you are right. But it will be picked up anyway, no matter in which order we do the checks. I think I need to remove the title from the pattern after it has been matched.

Reply:
Done.

Reply:
Huh. This seems overly complicated. I assume that we expect options first, then the title separated by a pipe?

Why not simply  `[$params, $title]` = sexplode('|', $params, 2);`?